client/src/view/tensegrity-view.tsx

```diff
@@ -148,6 +148,24 @@ export function TensegrityView({engine, fabricKernel, physics}:
             case "x":
                 connectClosestFacePair(fabric)
                 break
+            case "D":
+                const csvJoints: string[][] = []
+                const csvIntervals: string[][] = []
+                const output = fabric.output
+                csvJoints.push(["index", "x", "y", "z"])
+                output.joints.forEach(joint => {
+                    csvJoints.push([joint.index, joint.x, joint.y, joint.z])
+                })
+                csvIntervals.push(["joints", "type"])
+                output.intervals.forEach(interval => {
+                    csvIntervals.push([`"=""${interval.joints}"""`, interval.type])
+                })
+                console.log("Joints =======================\n", csvJoints.map(a => a.join(";")).join("\n"))
+                console.log("Intervals =======================\n", csvIntervals.map(a => a.join(";")).join("\n"))
+                break
+            case "d":
+                console.log(JSON.stringify(fabric.output, undefined, 4))
```

Comment:
lol, pretty ghetto way of getting the JSON out

Reply:
Build a server-side for it! We'll need it soon when we launch pretenst.com.